snuba/consumer.py

```diff
@@ -15,9 +15,11 @@
 
 logger = logging.getLogger("snuba.consumer")
 
-KafkaMessageMetadata = collections.namedtuple(
-    "KafkaMessageMetadata", "offset partition"
-)
+
+class KafkaMessageMetadata(NamedTuple):
+    offset: int
+    partition: int
+    timestamp: datetime
```

Comment:
Eventually, I'd like to phase this out and just pass `Message[TPayload]` throughout all the consumer machinery but that's not something I want to bite off right now and this already exists.

Reply:
fyi, I tried when I did this https://github.com/getsentry/snuba/pull/862
It is possible and not very hard. Just really verbose.

Reply:
Cool, I added it to the big list so it doesn't get lost.